src/main/features/response/routes/eligibility/send-your-response-by-email.ts

```diff
@@ -0,0 +1,33 @@
+import * as express from 'express'
+
+import { Paths } from 'response/paths'
+
+import { ResponseDraft } from 'response/draft/responseDraft'
+import { Draft } from '@hmcts/draft-store-client'
+import { FeesClient } from 'fees/feesClient'
+import { Range } from 'fees/models/range'
+
+const supportedFeeLimitInPennies: number = 1000000
+
+async function renderView (res: express.Response, next: express.NextFunction) {
```

Comment:
I guess in that particular case we do not need `renderView` function, we could inline it.